Gemfile

```diff
@@ -0,0 +1,3 @@
+source ENV['GEM_SOURCE'] || "https://rubygems.org"
```

Comment:
Not sure I like ruby stuff leaking out of the server folder. While Puppet is ruby, vscode is not. I wonder if we should use the tooling the community would expect